content/en/docs/tutorials/configuration/kubelet-standalone.md

```diff
@@ -0,0 +1,393 @@
+
+Install:
+
+```shell
+chmod +x kubelet
+mv kubelet /usr/bin/
+
+cat <<EOF > /etc/systemd/system/kubelet.service
+[Unit]
+Description=Kubelet
+
+[Service]
+ExecStart=/usr/bin/kubelet \
+ --config=/etc/kubernetes/kubelet.yaml
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+systemctl enable kubelet.service
+systemctl start kubelet.service
+```
+
+Quick test:
+
+```shell
+systemctl status kubelet.service
+```
+
+Detailed service check:
+
+```shell
+journalctl -u kubelet
+```
+
+## Run a static Pod in the Kubelet
+
+Configure:
+
+```shell
+cat <<EOF > static-web.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: static-web
+  labels:
+    role: myrole
+spec:
+  containers:
+    - name: web
+      image: nginx
+      ports:
+        - name: web
+          containerPort: 80
+          protocol: TCP
+EOF
+```
+
+Deploy:
+
+```shell
+mkdir -p /etc/kubernetes/manifests
+```
+
+Move the `static-web.yaml` manifest file to the `/etc/kubernetes/manifests` directory.
+
+```shell
+mv static-web-yaml /etc/kubernetes/manifests/
+```
+
+### Find out information about the Kubelet and the static Pod
+
+The cni-plugin creates a `veth` interface, retrieve the information:
+
+```shell
+ip link show | grep veth
+```
+
+The output is similar to:
+
+```
+6: veth85d4e050@if2: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP mode DEFAULT group default
+```
+
+Use the output to retrieve the IP address of the `veth` interface:
+
+```shell
+ip addr show veth85d4e050 | grep inet
+```
+
+The output is similar to:
+
+```
+    inet 10.241.1.1/32 scope global veth85d4e050
+    inet6 fe80::f04e:1ff:fefd:9464/64 scope link
+```
+
+Using the IP address of the veth interface, query the Kubelet's API endpoint at `http://10.241.1.1:10250/pods`:
+
+```shell
+curl http://10.241.1.1:10250/pods
+```
+
+Tip: Use `jq` to "prettify" the JSON output:
+
+```shell
+curl http://10.241.1.1:10250/pods | jq '.'
+```
+
+To obtain the IP address of the `nginx` Pod:
+
+```shell
+curl http://10.241.1.1:10250/pods | jq '.items[].status.podIP'
+```
+
+The output is similar to:
+
+```
+"10.241.1.11"
+```
+
+Connect to the `nginx` server on `http://<IP>:<Port>` (port 80 is the default), in this case:
+
+```shell
+curl http://10.241.1.11
+```
+
+The output is similar to:
+
+```
+<!DOCTYPE html>
+<html>
+<head>
+<title>Welcome to nginx!</title>
+...
+```
+
+Directories for monitoring and troubleshooting:
+
+```
+/var/lib/cni
+/var/lib/containerd
+/var/lib/kubelet
+
+/var/log/containers
+/var/log/pods
+```
+
+## Clean up
+
+Kubelet:
+
+```shell
+systemctl stop kubelet
+systemctl disable kubelet
+rm /usr/bin/kubelet
+rm -rf /etc/kubernetes
+rm -rf /var/lib/kubelet
+rm -rf /var/log/containers
+rm -rf /var/log/pods
+```
+
+Containerd:
+
+```shell
+systemctl stop containerd
+systemctl disable containerd
+rm /usr/local/bin/containerd
+rm -rf /etc/containerd
+rm -rf /var/lib/containerd
+```
+
+cni-plugins:
+
+```shell
+rm -rf /opt/cni
+rm -rf /etc/cni
+rm -rf /var/lib/cni
+```
+
```

Comment:
What about the systemd unit files; should we clean those up too?

Reply:
Done 

I would not use my personal or work computer to install system software as `root`, it's too risky. I would use a disposable system such as a virtual machine or a cloud instance for this tutorial.